java/ql/src/Security/CWE/CWE-079/AndroidWebViewSettingsEnabledJavaScript.qhelp

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE qhelp PUBLIC
+ "-//Semmle//qhelp//EN"
+ "qhelp.dtd">
+<qhelp>
+  <overview>
+<p>
+  Enabling JavaScript in an Android WebView allows for the running of JavaScript
+code in the context of the running application. This opens the possibility for a
+man-in-the-middle attack, where the attacker can inject arbitrary JavaScript.
+</p>
+
+<p>
+  You can enable or disbale Javascript execution using
+  the <code>setJavaScriptEnabled</code> method of the settings of a webview.
+</p>
+  </overview>
+
+  <recommendation>
+    <p>If Javascript does not need to be enabled, call <code>setJavaScriptEnabled(false)</code> on the settings of the webview.</p>
```

Comment:
Maybe mention that this is `false` by default, and thus it would also suffice to not explicitly enable it?